src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -15660,6 +15693,9 @@ public enum NormalizationForm
         public int EncodeToUtf8(System.Span<byte> destination) { throw null; }
         public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }
         public bool Equals(System.Text.Rune other) { throw null; }
+        public bool Equals(System.Text.Rune other, System.StringComparison comparisonType) { throw null; }
+        public static bool Equals(System.Text.Rune left, System.Text.Rune right) { throw null; }
+        public static bool Equals(System.Text.Rune left, System.Text.Rune right, StringComparison comparisonType) { throw null; }
         public override int GetHashCode() { throw null; }
```

Comment:
The review feedback suggested to not having static Equals and just include the instance overload.Just remove:

```csharp
        public static bool Equals(System.Text.Rune left, System.Text.Rune right) { throw null; }
        public static bool Equals(System.Text.Rune left, System.Text.Rune right, StringComparison comparisonType) { throw null; }
```